bin/s3-sync.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [[ -z "$1" ]]; then
+    echo "Bucket name required"
+    exit 1
+fi
+
+SHORT_CACHE="--cache-control max-age=60" # 1 minute
+LONG_CACHE="--cache-control max-age=604800" # 1 week
```

Comment:
@schalkneethling how long would you like this to be?

Reply:
I would feel comfortable with a month. If there is a new release of one of the dependency libs, the version number on the file will change and bust the cache right?

For our own files, 30 minutes would be ideal if possible, but could be longer if it makes more sense. Thanks!